src/services/ColorSchemeManager.ts

```diff
@@ -0,0 +1,21 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import type { StorageManager, ColorMode } from 'native-base'
+
+export const colorModeManager: StorageManager = {
+  get: async () => {
+    try {
+      const val = await AsyncStorage.getItem('@color-mode')
```

Comment:
There is a file named asyncStorageKeys, add it there. Also change this value to '@theme/native-base-color-mode', because we have color sheme in our app and it can be hard to find difference between this two

Reply:
Imo we can use the same key for NB color scheme manager as they point to exact same value. They depends on each other